src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string
+};
+
+export class App extends React.Component<{}, State> {
+  state: State = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKey);
+  }
+
+  handleKey = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: `${event.key}` });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <a href="/" className="App__message">
```

Comment:
good job but why did you change 'p' to  'a'?

Reply:
I have the same question: after clicking on this link, the site shows an error, so it should be fixed.